cli/src/test/java/org/commonjava/maven/ext/cli/CliTest.java

```diff
@@ -185,83 +187,210 @@ public void checkLocalRepositoryWithDefaultsAndModifiedUserSettings() throws Exc
+                                                                     + "org.eclipse.aether:aether-api:1.1.0                                             jar                 provided            \n"
+                                                                     + "org.eclipse.jetty:jetty-server:9.4.17.v20190418                                 jar                 compile             \n"
+                                                                     + "org.eclipse.sisu:org.eclipse.sisu.plexus:0.3.3                                  jar                 compile             \n"
+                                                                     + "org.hamcrest:hamcrest-all:1.3                                                   jar                 test                \n"
+                                                                     + "org.jacoco:jacoco-maven-plugin:0.8.5                                            maven-plugin                            \n"
+                                                                     + "org.jboss.byteman:byteman-bmunit:4.0.4                                          jar                 test                \n"
+                                                                     + "org.jboss.da:reports-model:1.7.0                                                jar                 compile             \n"
+                                                                     + "org.jdom:jdom:1.1.3                                                             jar                 compile             \n"
+                                                                     + "org.projectlombok:lombok:1.18.12                                                jar                 provided            \n"
+                                                                     + "org.projectlombok:lombok-maven-plugin:1.18.12.0                                 maven-plugin                            \n"
+                                                                     + "org.slf4j:slf4j-api:1.7.30                                                      jar                 compile             \n"
+                                                                     + "org.xmlunit:xmlunit-core:2.1.1                                                  jar                 test                \n"
+                                                                     + "org.xmlunit:xmlunit-matchers:2.1.1                                              jar                 test                \n"
+                                                                     + "org.yaml:snakeyaml:1.17                                                         jar                 compile             \n" ) );
+
+    }
+
+    @Test
+    public void checkManipulatorOrder()
```

Comment:
It makes this test a little hard to update when adding a new manipulator.

Reply:
Well, its kind of on purpose :-)

Reply:
I realize it has to be hardcoded, but it depends if this test is really is meant to test the particular order or just that the output works.

If you wanted to verify the relative ordering of the manipulators, I assume that is another test, and if you just want to verify that output works, it could (potentially) be made less strict. But, it's not a big deal.

Reply:
I think a bit of both - if we accidentally changed the order this would catch it, and this also tests the output.